site/kubernetes/operator/using-operator.md

```diff
@@ -530,6 +531,8 @@ This certificate must be stored in a Secret of name `spec.tls.caSecretName`, in
 object. Note that this can be the same Secret as `spec.tls.secretName`. This Secret **must** have a key `ca.crt` containing
 the CA certificate.
 
+RabbitMQ supports tls certificate rotation (same CA) without restarting itself. To rotate your tls certificate, you can simply update certificates in your tls Secret object. Updates to the Secret object will be picked up by the RabbitMQ pods within several minutes and you won't need to restart the RabbitmqCluster yourself.
```

Comment:
`same CA` -> I think more like `except CA`. CA cert is not even required for TLS unless it is mTLS
`restarting itself` -> `restarting the cluster`
`tls` -> `TLS`

Reply:
perhaps I am not clear enough about the CA business. What I'm trying to express is certificate rotation without restart is only supported if both the old and new certificate for the server is signed by the same CA. Is this wording "RabbitMQ supports TLS certificate rotation except CA without restarting the cluster" correct?